src/System.Management.Automation/FormatAndOutput/common/DisplayDatabase/displayDescriptionData.cs

```diff
@@ -788,7 +788,27 @@ public DisplayEntry(string value, DisplayEntryValueType type)
             ValueType = type;
         }
 
-        /// <summary/>
+        /// <summary>
+        /// Creates a DisplayEntry for the given scriptblock.
+        /// </summary>
+        /// <param name="scriptblock">The content of the scriptblock.</param>
+        /// <returns>A <see cref="DisplayEntry"/> for the <paramref name="scriptblock"/>.</returns>
+        public static DisplayEntry CreateScriptBlockEntry(string scriptblock)
+        {
+            return new DisplayEntry(scriptblock, DisplayEntryValueType.ScriptBlock);
```

Comment:
@PaulHigin should review this new public api for any security implications in creating script blocks that might circumvent the language mode.

Reply:
@PaulHigin reviewed this and told me he doesn't see a security issue.  @PaulHigin can you confirm?